sched/Kconfig

```diff
@@ -1647,6 +1647,13 @@ config SIG_POLL
 		The SIGPOLL signal is sent to a process when an asynchronous I/O
 		event occurs (meaning it has been polled).
 
+config SIG_SYS
+	int "SIGSYS"
+	default 30
```

Comment:
let's default to 31 to align with Linux's value

Reply:
I was thinking about that, but then we need to set `SIGWORK` default to 30.